bento_beacon/endpoints/info.py

```diff
@@ -128,11 +129,11 @@ def ga4gh_service_info():
     return {
         "id": current_app.config["BEACON_ID"],
         "name": current_app.config["BEACON_NAME"],
-        "type": {"artifact": "Beacon v2", "group": "org.ga4gh", "version": current_app.config["BENTO_BEACON_VERSION"]},
+        "type": {"artifact": "Beacon v2", "group": "org.ga4gh", "version": __version__},
```

Comment:
this version is usually expected to match the spec version I think (if we use ga4gh as the group and not specify it as a bento-type service), but that could be in a future PR

Reply:
oof